documentation/config/sample_vitess_connector_config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "sample_connector",
+  "config": {
+    "connector.class": "io.debezium.connector.vitess.VitessConnector",
+    "tasks.max": 1,
+    "vitess.keyspace": "commerce",
+    "vitess.shard": "0",
+    "vitess.vtgate.host": "host.docker.internal",
```

Comment:
Can we use the standard names known from other DBZ connectors: "database.host", "database.port"? It only will go so far, as there's unique concepts, like "keyspace", but where we can use common terminology, that'd be preferable.

Reply:
I also thought about it. Sure. We can use the common DBZ terminology whenever possible.